content/features/draft-and-publish.md

```diff
@@ -0,0 +1,33 @@
++++
+date = "2017-06-12T23:20:47-04:00"
+draft = false
+title = "New labs feature: Draft and Publish"
+description = "Prepare and review changes before pushing them to clients"
```

Comment:
Put a little more content/thought here. This is what shows up in the link preview, as well as in the table of contents in the blog.